### PR TITLE
variable-tabs: tab-size option and tab to spaces replacement

### PR DESCRIPTION
**Pull request by zxqfox**
I think we can swap all 4 spaces with tabs and replace them back on save.

With that technique we can provide to users any tab size they want in generated code.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -20,7 +20,12 @@ var BemGenerator = yeoman.generators.Base.extend({
         this.option('skip-install', {
             desc: 'Skip the installation of dependencies and libraries after generation of the project',
             type: Boolean,
-            required: 'false'
+            required: false
+        });
+        this.option('tab-size', {
+            desc: 'Skip the installation of dependencies and libraries after generation of the project',
```
Comment (tadatuta) [inline app/index.js]: copypaste ;)

File: app/index.js [hunk 2]
```diff
@@ -20,7 +20,12 @@ var BemGenerator = yeoman.generators.Base.extend({
         this.option('skip-install', {
             desc: 'Skip the installation of dependencies and libraries after generation of the project',
             type: Boolean,
-            required: 'false'
+            required: false
+        });
+        this.option('tab-size', {
+            desc: 'Tab size of the generated code in spaces. Specify 0 to generate tabs instead of spaces',
```
Comment (zxqfox) [inline app/index.js]: @tadatuta Thanks. Please check description

Comment (tadatuta) [inline app/index.js]: lgtm

Comment (zxqfox) [inline app/index.js]: Thanks. Rebased.

**Comment by eGavr** [comment]
Thanks :+1:

**Pull request merged**